lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -330,6 +333,37 @@ func (handler *StorageHandlersImpl) VolumeJoin(params storage.VolumeJoinParams)
 
 //utility functions
 
+// convert an SPL Image to a swagger-defined Image
```

Comment:
Not sure why it thinks I am adding convert image in this PR. Was already in the history set.